yt_dlp/extractor/zaiko.py

```diff
@@ -0,0 +1,55 @@
+import json
+import html
+import re
+
+from .common import InfoExtractor
+from ..utils import traverse_obj
+
+
+class ZaikoIE(InfoExtractor):
+    _VALID_URL = r'https://zaiko\.io/event/(?P<id>\d+)/stream(?:/\d+)+'
+    _TESTS = [{
+        'url': 'https://zaiko.io/event/324868/stream/20571/20571',
+        'info_dict': {
+            'id': 324868,
+            'ext': 'mp4',
+            'title': 'ZAIKO STREAMING TEST',
+            'uploader_id': 454,
+            'timestamp': 1583809200,
+        }
+    }]
+
+    def _parse_vue_element_attr(self, name, string):
+        page_elem = self._search_regex(r'<' + name + '([^>]+)>', string, name)
+        attr = {}
+        for m in re.finditer(r'(:?[\w\-_]+)="([^"]+)"', page_elem):
+            key, value = m.groups()
+            if key[0] == ':':
+                key = key[1:]
+                value = json.loads(html.unescape(value))
+            attr[key] = value
+        return attr
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        webpage = self._download_webpage(url, video_id)
+        stream_meta = self._parse_vue_element_attr('stream-page', webpage)
+
+        headers = {'referer': 'https://zaiko.io/'}
+        player_page = self._download_webpage(stream_meta['stream-access']['video_source'], video_id, headers=headers)
+        player_meta = self._parse_vue_element_attr('player', player_page)
```

Comment:
Just eliminating a one-time use variable and adding `video_id` args to the `_parse_vue` calls
```suggestion
        stream_meta = self._parse_vue_element_attr('stream-page', webpage, video_id)

        player_page = self._download_webpage(
            stream_meta['stream-access']['video_source'], video_id,
            headers={'referer': 'https://zaiko.io/'})
        player_meta = self._parse_vue_element_attr('player', player_page, video_id)
```